src/components/TokenSymbol/index.tsx

```diff
@@ -0,0 +1,23 @@
+import React, { useEffect, useState } from 'react'
+import { setImageToPlaceholder } from '../../routes/safe/components/Balances/utils'
+import Img from '../layout/Img'
+import { getTokenInfos } from '../../logic/tokens/store/actions/fetchTokens'
+import { Token } from '../../logic/tokens/store/model/token'
+
+type Props = {
+  tokenAddress: string
+}
+
+export const TokenSymbol = ({ tokenAddress }: Props): React.ReactElement => {
+  const [token, setToken] = useState<Token | null>(null)
+  useEffect(() => {
+    const fetchTokenInfo = async () => {
+      const tokenInfo = await getTokenInfos(tokenAddress)
+      setToken(tokenInfo)
+    }
+    fetchTokenInfo()
+  }, [tokenAddress])
```

Comment:
can we move this logic to a separate hook? `useToken` or something, so the usage will be the following
```
const token = useToken(tokenAddress)
```